entries/date-method.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <entry name="date" type="method" return="Boolean">
-	<title>date method</title>
+	<title>[DEPRECATED] date method</title>
 	<desc>Makes the element require a date.</desc>
 	<longdesc>
 		Return true if the value is a valid date. Uses JavaScript's built-in Date to test if the date is valid, and therefore does no sanity checks. Only the format must be valid, not the actual date, eg 30/30/2008 is a valid date.
-		<p>This method should not be used, since it relies on the <code>new Date</code> constructor, which behaves very differently across browsers and locales. Use <code>dateISO</code> instead or one of the locale specific methods (in localizations/ and additional-methods.js).</p>
+		<p><strong>DEPRECATION warning:</strong></p>
+		<p style="font-style: italic;"><strong>This method is deprecated and will be removed in version <code>2.0.0</code>.<br/>Please don't use it, since it relies on the <code>Date</code> constructor, which behaves very differently across browsers and locales. Use <code>dateISO</code> instead or one of the locale specific methods (in localizations/ and additional-methods.js).</strong></p>
```

Comment:
@staabm 

Should I extend this explanation with some examples?
IMHO, I think this message suffice.